bugbug/similarity.py

```diff
@@ -62,30 +52,40 @@
         duplicates[dupe].add(bug["id"])
 
 
-def get_text(bug):
-    return "{} {}".format(bug["summary"], bug["comments"][0]["text"])
-
-
-def text_preprocess(text, join=False):
-    for func in cleanup_functions:
-        text = func(text)
+class BaseSimilarity:
+    def __init__(self, cleanup_urls=True):
+        self.cleanup_urls = cleanup_urls
+
+    def get_text(self, bug):
+        return "{} {}".format(bug["summary"], bug["comments"][0]["text"])
+
+    def text_preprocess(self, text, join=False):
+        cleanup_functions = [
```

Comment:
Better to define these once in __init__ and reuse them here, otherwise you'll rerun the constructors of the cleanup functions for each text